Concepts/Tic-Tac-Toe/Entities/Board.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tic_Tac_Toe.Services.Interfaces;
+
+namespace Tic_Tac_Toe.Entities
+{
+    public class Board
+    {
+        private char[,] _grid;
```

Comment:
You can create a iterator in Board to return cells of grid one by one. This will keep us from exposing the grid directly.

Reply:
I am not able to create iterate so i need to discuss more about it